### PR TITLE
[Backend] Choices

### PR DESCRIPTION
**Pull request by brixtuyor**
# Related Links
- https://app.asana.com/0/1202403928837322/1202499306603308/f
- http://127.0.0.1:8000/lessons/:id

# Description
- End to end feature where a set of choices are displayed in the frontend for specific lessons

# Notes
- New migration added, please run `php artisan migrate`
- New seeder added, please run `php artisan db:seed --class=ChoiceSeeder`

# Screenshots
image
image

### CODE REVIEW

File: resources/js/Components/LessonCard.jsx [hunk 1]
```diff
@@ -1,31 +1,20 @@
-const LessonCard = ({ wordInJapanese, romanizedWord }) => {
+import React from "react";
+
+const LessonCard = (props) => {
```
Comment (roseaugusto) [inline resources/js/Components/LessonCard.jsx]: use destructuring here @brixtuyor so it's more cleaner

Comment (brixtuyor) [reply resources/js/Components/LessonCard.jsx]: Oh I forgot to edit it back, I was working on something when I did that. Will destructure it now...

Comment (brixtuyor) [reply resources/js/Components/LessonCard.jsx]: Done

**Comment by roseaugusto** [comment]
Fix weird Behavior
https://user-images.githubusercontent.com/70557829/177679222-e62823bf-c131-4741-b687-82628b042037.mov

**Comment by brixtuyor** [comment]
> Fix weird Behavior https://user-images.githubusercontent.com/70557829/177679222-e62823bf-c131-4741-b687-82628b042037.mov

Fixed
image

**Review by roseaugusto — APPROVED** [review]
Approved!

**Pull request merged**